mantidproject/mantid
# workbench about dialog

**Description of work.**
Add a Help-> About dialog that combines the functionality of the Help->about and first use dialogs from Mantidplot.


**To test:**
1. Check the links work
1. Check it formats ok on linux (and ideally mac)
1. Check that the do not open until next release works, you can edit the last_version in the workbench.ini to fool it.
1. Check that changing the usage works as it did in mantidplot.

Fixes #28481 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
